R/dominantTaxa.R

```diff
@@ -64,16 +70,16 @@ NULL
 #' @export
 setGeneric("perSampleDominantFeatures",signature = c("x"),
            function(x, assay.type = assay_name, assay_name = "counts", 
-                    rank = NULL, ...)
+                    rank = NULL, other_name = "Other", n = NULL, complete = FALSE, ...)
```

Comment:
is this needed? I thought it would work already with "..." but maybe I am mistaken.

Also, "other.name" would be preferred over "other_name" but perhaps that should be another PR later when we harmonize argument naming conventions.

Reply:
It should work without `n`, but I think it is more clear when it is explained here